apps/server/src/shared/domain/domainobject/user.do.ts

```diff
@@ -19,6 +19,8 @@ export class UserDO extends BaseDO {
 
 	ldapDn?: string;
 
+	legacyExternalId?: string;
```

Comment:
is legacy the correct word here? Is it actually an id that is exclusively for legacycode?

I think this is the externalid the user had before the migration. maybe something like
"migratedFromExternalId"? or think about it some more

Reply:
We want to obtain the legacyExternalId for the first iteration, we can still refactor that later. We will discuss that also in our team :)